contrib/testsuites/run-kvm-unit-test.sh

```diff
@@ -34,12 +34,12 @@ while [ true ]; do
             WILDCARD="$1"
             ;;
         "-h"|"--help")
-            echo "Usage: $0 [-h] [--endian ENDIAN] [--path PATH] [--wildcard WILDCARD] [avocado arguments ...]"
+            echo "Usage: $0 [-h] [--configure-args ['--endian ENDIAN][kvm-unit-test configure arguments]...'] [--path PATH] [--wildcard WILDCARD] [avocado arguments ...]"
```

Comment:
This line is corrupted. It should be:

```
Usage: $0 [-h] [--configure-args KVM_UNIT_TEST_CONFIGURE_ARGS] [--path PATH] [--wildcard WILDCARD] [avocado arguments ...]
```

or simply `[--configure-args CONFIGURE_ARGS]`, people can get the details from the help below...